src/main/java/slimeknights/mantle/client/book/data/content/ContentTextImage.java

```diff
@@ -41,4 +43,19 @@ public void build(BookData book, ArrayList<BookElement> list, boolean rightSide)
       list.add(new ImageElement(0, y + BookScreen.PAGE_HEIGHT - 100, BookScreen.PAGE_WIDTH, 100 - y, ImageData.MISSING));
     }
   }
+
+//  @Override
+//  public String toHTML() {
```

Comment:
Any reason this is commented out?

I'll admit this page does need some updates to make it better mirror `ContentImageText`, as I was noticing some weirdness when I used it in Metalborn.